### PR TITLE
Docs: Fix sample code of Block Preview in block registration docs

### PR DESCRIPTION
**Pull request by naogify**
## Description
Fix example code of adding innerBlocks to Block Preview in block registration document.
 I described the detail in #23898 

## How has this been tested?
I checked the preview of the document on my GitHub.
https://github.com/naogify/gutenberg/blob/docs/update-block-registration/docs/designers-developers/developers/block-api/block-registration.md

## Screenshots 
screenshot

## Types of changes
Documentation

## Checklist:
- [x] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: docs/designers-developers/developers/block-api/block-registration.md [hunk 1]
```diff
@@ -195,15 +195,15 @@ example: {
...
     },
-    innerBlocks: {
+    innerBlocks: [
         name: 'core/paragraph',
         attributes: {
             /* translators: example text. */
             content: __(
                 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent et eros eu felis.'
             ),
         },
```
Comment (ajitbohra) [inline docs/designers-developers/developers/block-api/block-registration.md:205]:
```suggestion
    {
        name: 'core/paragraph',
        attributes: {
            /* translators: example text. */
            content: __(
                'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent et eros eu felis.'
            ),
        }
    },
```

inner blocks array should be block objects.

**Review by ajitbohra — CHANGES_REQUESTED** [review]
inner blocks array should contain block object

**Comment by naogify** [comment]
@ajitbohra Thank you :)  I commited your change.

File: docs/designers-developers/developers/block-api/block-registration.md [hunk 2]
```diff
@@ -195,15 +195,17 @@ example: {
...
+    {
         name: 'core/paragraph',
         attributes: {
             /* translators: example text. */
             content: __(
                 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent et eros eu felis.'
             ),
-        },
+        }
     },
```
Comment (talldan) [inline docs/designers-developers/developers/block-api/block-registration.md:207]: This object (lines 199-207) should be indented by four spaces.

File: docs/designers-developers/developers/block-api/block-registration.md [hunk 3]
```diff
@@ -195,15 +195,17 @@ example: {
...
+    innerBlocks: [
+    {
         name: 'core/paragraph',
         attributes: {
             /* translators: example text. */
             content: __(
                 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent et eros eu felis.'
             ),
-        },
+        }
```
Comment (talldan) [inline docs/designers-developers/developers/block-api/block-registration.md:206]: Just missing a trailing comma here

**Review by talldan — COMMENTED** [review]
Sorry to request more changes.

Just a couple of things to make this match the coding standards.

File: docs/designers-developers/developers/block-api/block-registration.md (hunk 3 above)
Comment (naogify) [reply docs/designers-developers/developers/block-api/block-registration.md]: Thank you. I fixed it.

File: docs/designers-developers/developers/block-api/block-registration.md (hunk 2 above)
Comment (naogify) [reply docs/designers-developers/developers/block-api/block-registration.md]: Thank you @talldan. 

I just have a question.  According to the `.editorconfig` of Gutenberg, it says indent should be tab.
[https://github.com/naogify/gutenberg/blob/docs/update-block-registration/.editorconfig#L14](https://github.com/naogify/gutenberg/blob/docs/update-block-registration/.editorconfig#L14)

The codingstandards of of documentaiotn is different from the one?

**Review by mkaz — APPROVED** [review]
Thanks! 👍

File: docs/designers-developers/developers/block-api/block-registration.md (hunk 2 above)
Comment (talldan) [reply docs/designers-developers/developers/block-api/block-registration.md]: @naogify That's true, the coding standards say tabs - https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing.

In my comment above I mostly opted for consistency with the surrounding code which seemed to be indented using four spaces, but checking the rest of the file it seems inconsistent, sometimes tabs, sometimes spaces.

I think it probably doesn't matter so much in documentation, as long as it looks correct. The trouble is that it's harder to enforce without code linting.

Comment (mkaz) [reply docs/designers-developers/developers/block-api/block-registration.md]: @talldan We have documentation linting and prettier for docs, not enforced yet as the config and definitions needs some refining, but the basics are in place.

Editor config for setting up Prettier:
https://developer.wordpress.org/block-editor/contributors/document/#editor-config

And the linting uses markdownlint that also can be setup in editor or run using: `npm run lint:md-docs`

Comment (talldan) [reply docs/designers-developers/developers/block-api/block-registration.md]: @mkaz Thanks for the info! I managed to get that working in my editor. Looking forward to being able to write everything on one line, and then magically have it formatted when saving 😄 .

**Comment by mkaz** [comment]
@ajitbohra ping, I think this one is all set to merge, can you clear your "request change"? Thanks!

**Review by ajitbohra — APPROVED** [review]

File: docs/designers-developers/developers/block-api/block-registration.md (hunk 2 above)
Comment (naogify) [reply docs/designers-developers/developers/block-api/block-registration.md]: @mkaz @talldan 
Thank you for the information,and reply. I understand and will try the markdown linting :)

**Pull request merged**